Makefile

```diff
@@ -2,12 +2,14 @@
 quality:
```

Comment:
Maybe it would be nice to add `.PHONY` on top of this Makefile: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

```Makefile
.PHONY: quality style test test-common test-tf test-torch docs-single-version docs
```

Reply:
I agree nice idea :+1: